frontends/web/src/i18n/config.test.tsx

```diff
@@ -16,11 +16,16 @@
 
 import { describe, expect, it, Mock, vi } from 'vitest';
 vi.mock('./i18n');
-vi.mock('../utils/request');
+
+vi.mock('../utils/request', () => ({
+  apiGet: vi.fn().mockResolvedValue(null),
+}));
+
```

Comment:
You'll see this in several test files. **Without it, the tests still pass** but in my console at least I can see "error" about apiGet not defined. 

I'm trying to define it globally in the test setup file but still not succeeding. So I'm putting it in each file that needs it for now.

Reply:
I don't mind if we have it in each file where needed.